package.json

```diff
@@ -32,6 +33,7 @@
     "encrypt": "ts-node encryptor/encrypt.ts"
   },
   "devDependencies": {
+    "@types/jsonwebtoken": "^9.0.1",
```

Comment:
I’d also recommend removing the caret (^) from the dependency version for consistency and to lock the version explicitly.

Reply:
got it!